### ISSUE TITLE
include isn't working with Salt 3006

### ISSUE THREAD

**Issue by jpmckinney** [open-contracting/deploy#432]
`{% include 'apache/includes/' + name + '.include.jinja' ignore missing %}`

We have:

```yaml
  extra_filerefs:
    - $REPO_DIR/salt/apache/includes/cove.include.jinja
```

But `ErrorDocument 500` isn't getting written.

**Comment by jpmckinney** [comment]
The same is happening for me with `{% include 'postgres/files/conf/shared.include' %}`

I tried:

* passing `--extra-filerefs` both with absolute and `salt://` paths
* using absolute path in `include` 
* changing to `extends`, adding a block in `shared.include` and using that block in the conf files
* using absolute path in `extends`
* Removing `extra_filerefs` from Saltfile and repeating some of the above

Nothing seems to work.

**Comment by jpmckinney** [comment]
Another option is to use `shared.include` as the template, and then provide the custom settings (`max_wal_size`, etc.) as Pillar data. (Basically, reverse the direction of the inclusion.)

I'll do this because Salt seems incapable of keeping extra-filerefs working across versions (not the first time it breaks).

**Comment by jpmckinney** [comment]
Closed by 2048551c6bf975c68189634e0a0c995daf9e5f51
